src/codegen/utilities/interpreter_helpers.py

```diff
@@ -43,6 +42,10 @@
     "write_raw",
 ]
 
+INCLUDE_SIZE_PARAMETER_IN_SIGNATURE_FUNCTIONS = [
+    "get_analog_power_up_states_with_output_type",
```

Comment:
I think this is a special case because the metadata is wrong.

The size is passed by pointer:
```
int32 __CFUNC_C DAQmxGetAnalogPowerUpStatesWithOutputType(const char channelNames[], float64 stateArray[], int32 channelTypeArray[], uInt32 *arraySizePtr);
```

But the metadata says direction:in and doesn't say pointer:True.
https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/functions.py#L12829
```
            {
                'ctypes_data_type': 'ctypes.c_uint32',
                'direction': 'in',
                'name': 'arraySize',
                'python_data_type': 'int',
                'type': 'uInt32'
            }
```

Reply:
If you want to handle this in a separate PR, that's okay.

Reply:
As you have mentioned, this is a special case and we have manually handled in scrapigen metadata generation in transform, hence updated the direction in this [PR](https://github.com/ni/grpc-device-scrapigen/pull/168). (Only updated the direction since pointer flag is not used and out is used pass as pointer. Let us know if you feel that is also essential in metadata)